google/cloud/security/inventory/pipelines/load_group_members_pipeline.py

```diff
@@ -102,16 +104,37 @@ def _retrieve(self):
                 raise inventory_errors.LoadDataPipelineError(e)
 
             group_members_map.append((group_id, group_members))
+            LOGGER.debug('Retrieved members from %s: %d',
+                         group_id,
+                         len(group_members))
 
         return group_members_map
 
     def run(self):
         """Runs the load GSuite account groups pipeline."""
-        groups_members_map = self._retrieve()
-
-        if isinstance(groups_members_map, list):
-            loadable_group_members = self._transform(groups_members_map)
-            self._load(self.RESOURCE_NAME, loadable_group_members)
-            self._get_loaded_count()
-        else:
-            LOGGER.warn('No group members retrieved.')
+
+        group_ids = self._fetch_groups_from_dao()
+
+        def chunker(seq, size):
+            """Helper to chunk a list.
+
+            Args:
+                seq: A list.
+                size: Integer of the desired chunk size.
+
+            Returns:
+                A tuple of the chunked seq.
+            """
+            return (seq[pos:pos + size] for pos in xrange(0, len(seq), size))
+
+        # TODO: keep track of group_ids that are not retrieved/committed to db
+        for group_ids_chunk in chunker(group_ids, GROUP_CHUNK_SIZE):
+            LOGGER.debug('Retrieving a batch of group members')
```

Comment:
nit: It'd be helpful to put what our chunk size is and where we are in the total set of chunks so far in the debugging statement.